integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -196,6 +196,9 @@ fuse:
   jvmOptions: " -XX:MaxDirectMemorySize=2g "
   hostNetwork: true
   dnsPolicy: ClusterFirstWithHostNet
+  user: 0
+  group: 0
+  fsGroup: 0
```

Comment:
@madanadit I don't know much about fuse but does it only work with root access?

Reply:
yes, fuse container need as root with sys_admin privileges. it actually modifies the mount table on the physical host (highly privileged)